vhdbuilder/packer/generate-windows-vhd-configuration.ps1

```diff
@@ -153,8 +153,7 @@ $global:map = @{
     );
     "c:\akse-cache\calico\" = @(
         "https://acs-mirror.azureedge.net/calico-node/v3.20.0/binaries/calico-windows-v3.20.0.zip",
-        "https://acs-mirror.azureedge.net/calico-node/v3.21.0/binaries/calico-windows-v3.21.0.zip",
-        "https://acs-mirror.azureedge.net/calico-node/v3.21.2/binaries/calico-windows-v3.21.2.zip"
+        "https://acs-mirror.azureedge.net/calico-node/v3.21.2/binaries/calico-windows-v3.21.4.zip"
```

Comment:
```suggestion
        "https://acs-mirror.azureedge.net/calico-node/v3.21.4/binaries/calico-windows-v3.21.4.zip"
```

Reply:
Good catch, @sozercan! Always made mistakes here. Will improve it later.